packages/run-it/src/RunIt.tsx

```diff
@@ -126,6 +127,7 @@ export const RunIt: FC<RunItProps> = ({
     initRequestContent(configurator, inputs)
   )
   const [activePathParams, setActivePathParams] = useState({})
+  const [activeQueryParams, setActiveQueryParams] = useState({})
```

Comment:
suggestion: given how tightly coupled path and query params are and that they are always updated together, consider merging these into one state object, `activeRequestParams`. Resultant state updates can then happen in one call.